ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -44,6 +46,7 @@ class PillButtonBarDemoController: DemoController {
         let disablePrimarySwitchView = UISwitch()
         disablePrimarySwitchView.isOn = true
         disablePrimarySwitchView.addTarget(self, action: #selector(togglePrimaryPills(switchView:)), for: .valueChanged)
+        NSLayoutConstraint.activate([disablePrimarySwitchView.widthAnchor.constraint(equalToConstant: disablePrimarySwitchView.frame.width)])
 
         container.addArrangedSubview(createLabelWithText("Primary"))
         addRow(items: [createLabelWithText("Enable/Disable pills in Primary Pill bar"), disablePrimarySwitchView], itemSpacing: 20, centerItems: true)
```

Comment:
I agree with @harrieshin's comment that disabling the UISwitch constraints might not be the right fix here.
The container stack view is expanding to the point that it's not respecting the correct margins: 

![Screen Shot 2022-01-04 at 9 04 24 AM](https://user-images.githubusercontent.com/68076145/148096536-2aa0faa7-64c2-45c8-906e-06a03ad2f876.png)

I think that the stackview should ensure that it stays within the right margins and the label view should be flexible in its width while truncating the text if needed.